packages/wrangler/src/__tests__/user.test.ts

```diff
@@ -101,16 +108,13 @@ describe("User", () => {
 		});
 	});
 
-	// TODO: Improve OAuth mocking to handle `/token` endpoints from different calls
 	it("should handle errors for failed token refresh", async () => {
 		setIsTTY(false);
 		writeAuthConfigFile({
 			oauth_token: "hunter2",
 			refresh_token: "Order 66",
 		});
-		mockExchangeRefreshTokenForAccessToken({
-			respondWith: "badResponse",
-		});
+		// TODO: Use MSW to handle `/token` endpoints from different calls
```

Comment:
Is this test passing because the refresh token endpoint is now 404ing?
Should we not have a clear mock here that returns a 200 response with `badResponse`?

Reply:
I can add inline mock.